<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Instrument:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="instrumentBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>391</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <item>
     <property name="text">
       <string>Acoustic Grand Piano</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Bright Acoustic Piano</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Electric Grand Piano</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Honky-tonk Piano</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Electric Piano 1</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Electric Piano 2</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Harpsichord</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Clavinet</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Celesta</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Glockenspiel</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Music Box</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Vibraphone</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Marimba</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Xylophone</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Tubular Bells</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Dulcimer</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Drawbar Organ</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Percussive Organ</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Rock Organ</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Church Organ</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Reed Organ</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Accordion</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Harmonica</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Tango Accordion</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Acoustic Guitar (nylon)</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Acoustic Guitar (steel)</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Electric Guitar (jazz)</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Electric Guitar (clean)</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Electric Guitar (muted)</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Overdriven Guitar</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Distortion Guitar</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Guitar Harmonics</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Acoustic Bass</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Electric Bass (finger)</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Electric Bass (pick)</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Fretless Bass</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Slap Bass 1</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Slap Bass 2</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Synth Bass 1</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Synth Bass 2</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Violin</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Viola</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Cello</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Contrabass</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Tremolo Strings</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Pizzicato Strings</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Orchestral Harp</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Timpani</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>String Ensemble 1</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>String Ensemble 2</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Synth Strings 1</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Synth Strings 2</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Choir Aahs</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Voice Oohs</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Synth Choir</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Orchestra Hit</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Trumpet</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Trombone</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Tuba</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Muted Trumpet</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>French Horn</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Brass Section</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Synth Brass 1</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Synth Brass 2</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Soprano Sax</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Alto Sax</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Tenor Sax</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Baritone Sax</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Oboe</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>English Horn</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Bassoon</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Clarinet</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Piccolo</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Flute</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Recorder</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Pan Flute</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Blown Bottle</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Shakuhachi</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Whistle</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Ocarina</string>
     </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Excercise:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Intervals:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="excerciseBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>391</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <item>
     <property name="text">
       <string>Melodic intervals</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Descending melodic intervals</string>
     </property>
   </item>
   <item>
     <property name="text">
       <string>Ascending melodic intervals</string>
     </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>634</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="interval0">
      <property name="text">
       <string>Perfect unison</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="interval1">
      <property name="text">
       <string>Minor second</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="interval2">
      <property name="text">
       <string>Major second</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="interval3">
      <property name="text">
       <string>Minor third</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="interval4">
      <property name="text">
       <string>Major third</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="interval5">
      <property name="text">
       <string>Perfect fourth</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="interval6">
      <property name="text">
       <string>Diminished fifth</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="interval7">
      <property name="text">
       <string>Perfect fifth</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="interval8">
      <property name="text">
       <string>Minor sixth</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="interval9">
      <property name="text">
       <string>Major sixth</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="interval10">
      <property name="text">
       <string>Minor seventh</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="interval11">
      <property name="text">
       <string>Major seventh</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="interval12">
      <property name="text">
       <string>Perfect Octave</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
